TheFourFifths/consus-client
# Student to rfid assocation

### Summary

You can now scan an rfid card and associate it with an existing student.

### Checklist


### How to Review

1. Start client on this branch and server on student-exists-tff207
2. In the omnibar, enter: 123456
3. Verify that John shows up(he is already associated with an rfid)
4. In the omnibar, enter: 147852
5. Verify input modal is up.
6, Enter studentID: 159753(meet Mr Rufflekins!)
7. Click associate button
8. Confirm associate toast appears
9. In the omnibar enter: 147852
10. Verify Mr. Rufflekins shows up.
11. Go to view all students page and confirm Mr. Rufflekins has an rfid associated.

### Links

- [TFF-207](https://msoese.atlassian.net/browse/TFF-207)
- TheFourFifths/consus/pull/102
